src/core/tests.rs

```diff
@@ -249,40 +250,49 @@ fn test_integrity_check() {
 }
 
 #[test]
-fn test_covercrypt_kem() {
-    let policy = policy().expect("cannot generate policy");
+fn test_covercrypt_kem() -> Result<(), Error> {
     let ap = AccessPolicy::parse("Department::FIN && Security Level::Top Secret").unwrap();
-    let cc = Covercrypt::default();
-    let (mut msk, _) = cc.setup().expect("cannot generate master keys");
+    let (mut msk, _mpk, cc) = msk()?;
```

Comment:
unwrap here (btw this function should be renamed as it does not only return a `msk`).
```suggestion
    let (mut msk, _mpk, cc) = msk().unwrap();
```